src/solvers/dgsem_tree/indicators_2d.jl

```diff
@@ -190,6 +190,231 @@ function (löhner::IndicatorLöhner)(u::AbstractArray{<:Any,4},
+      if cache.mortars.orientations[mortar] == 1
+        # L2 mortars in x-direction
+        index_large = i -> (nnodes(dg), i)
+        index_small = i -> (1, i)
+        element_solutions_to_mortars!(u_old, indicator_IDP, dg, equations,
+                                      large, upper, lower, index_large, index_small)
+      else
+        # L2 mortars in y-direction
+        index_large = i -> (i, nnodes(dg))
+        index_small = i -> (i, 1)
+        element_solutions_to_mortars!(u_old, indicator_IDP, dg, equations,
+                                      large, upper, lower, index_large, index_small)
+      end
+    else # large_sides[mortar] == 2 -> large element on right side
+      if cache.mortars.orientations[mortar] == 1
+        # L2 mortars in x-direction
+        index_large = i -> (1, i)
+        index_small = i -> (nnodes(dg), i)
+        element_solutions_to_mortars!(u_old, indicator_IDP, dg, equations,
+                                      large, upper, lower, index_large, index_small)
+      else
+        # L2 mortars in y-direction
+        index_large = i -> (i, 1)
+        index_small = i -> (i, nnodes(dg))
+        element_solutions_to_mortars!(u_old, indicator_IDP, dg, equations,
+                                      large, upper, lower, index_large, index_small)
+      end
+    end
+  end
+
+  @threaded for element in eachelement(dg, cache)
+    inverse_jacobian = cache.elements.inverse_jacobian[element]
+
+    for j in eachnode(dg), i in eachnode(dg)
+      u_local = get_node_vars(u, equations, dg, i, j, element)
+      var = indicator_IDP.variable(u_local, equations)
+      if abs(var_max[i, j, element] - var) < sqrt(eps()) || abs(var_min[i, j, element] - var) < sqrt(eps())
+        alpha[i, j, element] = 0.0
+        continue
+      end
+      # Calculate P_plus and P_minus
+      # Note: Boundaries of antidiffusive_flux1/2 are constant 0, so they make no difference here.
+      val_flux1_local     = indicator_IDP.variable( dt * inverse_jacobian * inverse_weights[i] * get_node_vars(antidiffusive_flux1, equations, dg, i,   j,   element), equations)
+      val_flux1_local_ip1 = indicator_IDP.variable(-dt * inverse_jacobian * inverse_weights[i] * get_node_vars(antidiffusive_flux1, equations, dg, i+1, j,   element), equations)
+      val_flux2_local     = indicator_IDP.variable( dt * inverse_jacobian * inverse_weights[j] * get_node_vars(antidiffusive_flux2, equations, dg, i,   j,   element), equations)
+      val_flux2_local_jp1 = indicator_IDP.variable(-dt * inverse_jacobian * inverse_weights[j] * get_node_vars(antidiffusive_flux2, equations, dg, i,   j+1, element), equations)
+
+      P_plus  = max(0.0, val_flux1_local) + max(0.0, val_flux1_local_ip1) +
+                max(0.0, val_flux2_local) + max(0.0, val_flux2_local_jp1)
+      P_minus = min(0.0, val_flux1_local) + min(0.0, val_flux1_local_ip1) +
+                min(0.0, val_flux2_local) + min(0.0, val_flux2_local_jp1)
```

Comment:
@amrueda On Friday, I spoke about the high indicator values on the boundaries and corners of an element. That seems to be right for my implementation of `P_plus` and `P_minus` since `antififfusive_flux` is zero at the boundaries and hence `P_plus/minus` are smaller according to amount. Is this intentional or have I misunderstood something?

Reply:
@bennibolm 
You are right. The antidiffusive flux should be zero at the boundaries since the FV and DG fluxes match there. That leads to smaller magnitudes of `P_plus` and `P_minus` at the boundaries. However, that does not explain right away why alpha is higher there, as `alpha = 1 - something/P_plusorminus`.

That being said, I have also observed in my simulations that the value of alpha is (in many cases) higher at the element boundaries than in the inner DOFs. That does not necessarily mean that something is wrong.

Reply:
> @bennibolm You are right. The antidiffusive flux should be zero at the boundaries since the FV and DG fluxes match there. That leads to smaller magnitudes of `P_plus` and `P_minus` at the boundaries. However, that does not explain right away why alpha is higher there, as `alpha = 1 - something/P_plusorminus`.

Oh yeah, that's right.
> That being said, I have also observed in my simulations that the value of alpha is (in many cases) higher at the element boundaries than in the inner DOFs. That does not necessarily mean that something is wrong.

Okay, thanks.